.github/workflows/tox_pytests.yml

```
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9, "3.10"]
+        python-version: [3.9, "3.10", "3.11"]
```

```suggestion
        python-version: [3.9, "3.10", "3.11", "3.12"]
```
We should definitely support the current stable Python version, which is 3.12. I don't know, if we should even continue supporting Python 3.9: Debian 12 (current stable) has Python 3.11 already, Ubuntu 22.04 has Python 3.10 as default.